### PR TITLE
Update contentType.schema.tpl.json

### PR DESCRIPTION
**Pull request by lzehl**
Slight changes in the contentType schema according to discussion in PR https://github.com/HumanBrainProject/openMINDS_instances/pull/2

### CODE REVIEW

File: schemas/data/contentType.schema.tpl.json [hunk 1]
```diff
@@ -15,9 +14,9 @@
         "type": "string"
       }
     },
-    "category": {
+    "description": {
       "type": "string",
-      "_instruction": "Enter the category to which this content type belongs to."
+      "_instruction": "Enter a description of the content type specification if no public specification file can be linked. Leave blank an d use 'specification' if a public specification can be linked."
```
Comment (apdavison) [inline schemas/data/contentType.schema.tpl.json]: I would say "Enter a description of the content type specification. May be left blank if a public specification can be linked in 'specification'"

File: schemas/data/contentType.schema.tpl.json [hunk 2]
```diff
@@ -28,6 +27,11 @@
       "type": "string",
       "_instruction": "Enter the name (iana-inspired convention) of this content type."
     },
+    "specification":{
+      "type": "string",
+      "format": "iri",
+      "_instruction": "Enter the iternationalized resource identifier (IRI) to the official specification of this content type. Leave blank and use 'description' to provide some specification if an official specification is not available."
```
Comment (apdavison) [inline schemas/data/contentType.schema.tpl.json]: "of the official"

**Comment by apdavison** [comment]
Looks good. Shall I merge? Are you ok with squash merge, to have this appear as a single commit?

**Comment by lzehl** [comment]
@apdavison yes please go ahead and merge. Squash merge is fine, and thanks for the quick review :)

**Pull request merged**